### PR TITLE
Removed if statement and moved logger.debug after updating the status

### PR DESCRIPTION
**Pull request by JAAlvarado-Montes**
Hi,

This small PR removes the if statement to update the mount status even while parked. Also, the logs are now written once the mount status has been updated.

Cheers,
Jaime.

### CODE REVIEW

**Comment by JAAlvarado-Montes** [comment]
Changes have been tested with hardware and TSX is able to report the mount status even while parked

File: src/panoptes/pocs/mount/bisque.py [hunk 1]
```diff
@@ -135,8 +134,8 @@ def _update_status(self):
         except KeyError:
             self.logger.warning("Problem with status, key not found")
 
-        if not self._is_parked:
-            status.update(self.query('get_coordinates'))
+        status.update(self.query('get_coordinates'))
+        self.logger.debug(f"Status: {status}")
```
Comment (danjampro) [inline src/panoptes/pocs/mount/bisque.py:138]: Maybe this would be better as `Mount status: {status}`?

**Review by danjampro — APPROVED** [review]
LGTM, maybe change status message.

**Comment by JAAlvarado-Montes** [comment]
Done!

**Pull request merged**